assets/src/edit-story/components/inspector/inspectorTabs.js

```diff
@@ -30,14 +30,14 @@ const Tab = styled.button.attrs( { role: 'tab' } )`
 	text-transform: uppercase;
 	color: ${ ( { theme } ) => theme.colors.bg.v4 };
 	font-weight: ${ ( { isActive } ) => isActive ? 'bold' : 'normal;' };
+	word-break: break-word;
```

Comment:
```suggestion
	word-break: break-word;
	padding: 0;
        font-size: 14px;
```
This fixed it for me

Reply:
Handled in 3a5e70ce08d4d73da52b183490a8f3cb7c100e53.